spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -388,8 +388,8 @@ describe('document-capture/components/acuant-capture', () => {
       const button = getByLabelText('Image');
       await userEvent.click(button);
 
-      await waitFor(() => !container.querySelector('.full-screen'));
-      expect(document.activeElement).to.equal(outsideInput);
+      await waitFor(() => document.activeElement === outsideInput);
+      expect(container.classList.contains('full-screen')).to.be.false();
```

Comment:
Probably another timing issue, since at the time the final assertion was being made, the active element wasn't (yet) the intended target. By switching the order, we wait for the expected focus element, and at that point we can still assume the full screen class is removed from the container.